cpp/data/installer/ub_tools.sql

```diff
@@ -39,11 +39,12 @@ CREATE TABLE zeder_journals (
 
 -- Table to be used w/ our validate_harvested_records tool:
 CREATE TABLE metadata_presence_tracer (
-    zeder_journal_id INT(11) UNSIGNED,
+    journal_id INT(11) UNSIGNED,
     marc_field_tag CHAR(3) NOT NULL,
+    marc_subfield_code CHAR(1) NOT NULL,
     record_type ENUM('regular_article', 'review') DEFAULT 'regular_article' NOT NULL,
     field_presence ENUM('always', 'sometimes', 'ignore') NOT NULL,
-    CONSTRAINT metadata_presence_tracer_zeder_journal_id FOREIGN KEY (zeder_journal_id) REFERENCES zeder_journals (id) ON DELETE CASCADE ON UPDATE CASCADE,
+    CONSTRAINT metadata_presence_tracer_zeder_journal_id FOREIGN KEY (journal_id) REFERENCES zeder_journals (id) ON DELETE CASCADE ON UPDATE CASCADE,
```

Comment:
The name of this constraint should be changed as well (also in ub_tools.24)

Reply:
Also, the second constraint still contains a reference to zeder_journal_id

Reply:
Fixed